packages/Akvo_rabbitmq_client/tests/test_queue_message_util.py

```diff
@@ -1,20 +1,68 @@
 import unittest
+
 from uuid import UUID, uuid4
 from Akvo_rabbitmq_client import queue_message_util
+from enum import Enum
+
+
+class ChatSenderEnum(Enum):
```

Comment:
Ok, I promise this is the final change I request: please rename the `sender` field to `sender_role` and rename the `ChatSenderEnum` to `ChatRoleEnum`. In my opinion that makes the intention of this field more clear.